### PR TITLE
docs: describe nfd-topology-gc in introduction.md

### PR DESCRIPTION
**Pull request by yanggangtony**
fix omit describe for nfd-topology-gc.

### CODE REVIEW

File: docs/get-started/introduction.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ NFD consists of three software components:
 1. nfd-master
 1. nfd-worker
 1. nfd-topology-updater
+1. nfd-topology-gc
```
Comment (marquiz) [inline docs/get-started/introduction.md:27]: Could you also adjust the `NFD consists of three software components:` text above

Also, add a new sub-section for "NFD-Topology-GC". Just a one sentence description is enough for now

**Review by marquiz — COMMENTED** [review]
Thanks @yanggangtony for the contribution

Could you add a `docs:` "topic" to the git commit message title, e.g.

`docs: describe nfd-topology-gc in introduction.md`

or smth...

**Comment by yanggangtony** [comment]
@marquiz 
Thanks for review.
Rebased it.

**Review by marquiz — APPROVED** [review]
Thanks @yanggangtony for the update. Looks good to me

I didn't realize that we already had a sub-section about topology-gc as it was named NFD-Topology-**Garbage-Collector** 🙈

/retitle docs: describe nfd-topology-gc in introduction.md

**Review by ffromani — COMMENTED** [review]
/lgtm

**Pull request merged**